ibm/service/power/ibm_pi_constants.go

```diff
@@ -6,11 +6,25 @@ const (
 	// Arguments
 	Arg_CloudConnectionName                 = "pi_cloud_connection_name"
 	Arg_CloudInstanceID                     = "pi_cloud_instance_id"
+	Arg_DatacenterZone                      = "pi_datacenter_zone"
+	Arg_DhcpCidr                            = "pi_cidr"
+	Arg_DhcpCloudConnectionID               = "pi_cloud_connection_id"
+	Arg_DhcpDnsServer                       = "pi_dns_server"
+	Arg_DhcpID                              = "pi_dhcp_id"
+	Arg_DhcpName                            = "pi_dhcp_name"
+	Arg_DhcpSnatEnabled                     = "pi_dhcp_snat_enabled"
+	Arg_IBMiCSS                             = "pi_ibmi_css"
+	Arg_IBMiPHA                             = "pi_ibmi_pha"
+	Arg_IBMiRDSUsers                        = "pi_ibmi_rds_users"
 	Arg_ImageName                           = "pi_image_name"
 	Arg_InstanceName                        = "pi_instance_name"
 	Arg_KeyName                             = "pi_key_name"
 	Arg_NetworkName                         = "pi_network_name"
+	Arg_PIInstanceSharedProcessorPool       = "pi_shared_processor_pool"
```

Comment:
This variable probably really needs to be relabeled, but the sorting is fine.

Reply:
Eventually all those legacy names will change.